handsontable/src/pluginHooks.js

```diff
@@ -1357,6 +1357,18 @@ const REGISTERED_HOOKS = [
    */
   'modifyColWidth',
 
+  /**
+   * Fired when focusing a cell or a header element. Allows replacing the element to be focused by returning a
+   * different HTML element.
+   *
+   * @since 14.0.0
+   * @event Hooks#modifyFocusedElement
+   * @param {number} row Row index.
+   * @param {number} column Column index.
+   * @param {HTMLElement} focusedElement The element to be focused.
```

Comment:
Missing docs about what the hook can return (HTMLElement or undefined).

Reply:
Done in https://github.com/handsontable/handsontable/pull/10342/commits/cd04737fe2df87362f40b2f431943dd173226189